textdb/textdb-web/src/main/java/edu/uci/ics/textdb/web/request/QueryPlanRequest.java

```diff
@@ -22,7 +23,9 @@
     private ArrayList<OperatorBean> operatorBeans;
     @JsonProperty("links")
     private ArrayList<OperatorLinkBean> operatorLinkBeans;
+    @JsonIgnore
     private HashMap<String, HashMap<String, String>> operatorProperties;
+    @JsonIgnore
     private LogicalPlan logicalPlan;
 
     public static final String GET_PROPERTIES_FUNCTION_NAME = "getOperatorProperties";
```

Comment:
I have a comment irrelevant to this PR: in the next line (line 32) can you change the two places where has raw type `Class` to `Class<? extends OperatorBean>`, so that the IDE won't complain about raw type?